src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/__init__.py

```diff
@@ -0,0 +1,111 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+import os
+from six.moves import input, configparser #pylint: disable=redefined-builtin
+from azure.cli.commands import cli_command
+import azure.cli._logging as _logging
+from azure.cli._config import GLOBAL_CONFIG_PATH, ACTIVE_ENV_CONFIG_PATH, ENV_VAR_PREFIX
+import azure.cli.command_modules.configure._help # pylint: disable=unused-import
+
+logger = _logging.get_az_logger(__name__)
+
+OUTPUT_LIST = [
+    {'name': 'json', 'desc': 'JSON formatted output that most closely matches API responses'},
+    {'name': 'jsonc', 'desc': 'Colored JSON formatted output that most closely matches API responses'}, #pylint: disable=line-too-long
+    {'name': 'table', 'desc': 'Human-readable output format that focuses on clarity'},
```

Comment:
Table revisions are part of PR #748 so once that gets merged, most commands will have table output.
The 'focuses on clarity' should indicate that not all fields will be present? Also, it'll be clear when the tables are displayed that it doesn't show everything.